types/htmlparser2/index.d.ts

```diff
@@ -63,31 +53,49 @@ export interface Options {
 }
 
 export declare class WritableStream extends Writable {
-    constructor(handler: Handler, options?: Options);
+    constructor(handler: DomHandler);
+    constructor(handler: DomHandler, options: ParserOptions);
 }
 
 export declare class Parser {
-    constructor(handler: Handler, options?: Options);
+    constructor(handler: DomHandler);
+    constructor(handler: DomHandler, options: ParserOptions);
```

Comment:
Splitting this up into two functions won't allow `new Parser(handler, undefined)`, was that intentional?

Reply:
Yes, this is wrong. I'll change!